Checkpointing: lbboundaries, constraints, auto_update_accumulators +test

RudolfWeeber opened the pull request.

RudolfWeeber committed:
Checkpointing: lbboundaries, constraints, auto_update_accumulators +test

RudolfWeeber committed:
Merge branch 'python' of git://github.com/espressomd/espresso into checkpointing

RudolfWeeber committed:
Revert erroneous changes to test case

jngrad commented:
I can't get the new check `test_lb_boundaries()` to pass in `test_checkpoint_ek.gpu_*`

jngrad committed:
Minor cleanup

Naming convention: LB and EK are boolean variables which depend on
modes enabled in a checkpointing test. Fix minor regressions from
the merge. Formatting.

jngrad committed:
Fix several issues found in CI

RudolfWeeber committed:
Merge branch 'python' of git://github.com/espressomd/espresso into checkpointing

RudolfWeeber committed:
Merge branch 'checkpointing' of ssh://github.com/RudolfWeeber/espresso into checkpointing

RudolfWeeber committed:
Checkpointing: LB boundaries cannot be used when EK is on, so don't try

RudolfWeeber committed:
Merge branch 'python' of git://github.com/espressomd/espresso into HEAD

RudolfWeeber committed:
np.copy

RudolfWeeber committed:
Testsuite: Fix test pre-condition for lb oundaires checkpointing

RudolfWeeber committed:
Merge branch 'python' of git://github.com/espressomd/espresso into checkpointing2

fweik commented:
bors r+

jngrad commented:
bors retry
